src/main/java/TaskList.java

```diff
@@ -0,0 +1,87 @@
+import java.lang.StringBuilder;
+import java.util.ArrayList;
+
+public class TaskList {
+    protected ArrayList<Task> tasks;
+
+    /**
+     *  Constructs a <code>TaskList</code> object that stores <code>Task</code> objects in an ordered list.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    /**
+     *  Retrieves a <code>Task</code> with a given task ID from this <code>TaskList</code>.
+     *  @param id <code>int</code> ID of the task.
+     *  @return the <code>Task</code> with the given ID.
+     */
+    public Task getTask(int id) {
+        return this.tasks.get(id - 1);
+    }
+
+    /**
+     *  Inserts a <code>Task</code> as the next entry in this <code>TaskList</code>.
+     *  @param task the <code>Task</code> to be inserted.
+     */
+    public void addTask(Task task) {
+        this.tasks.add(task);
+    }
+
+    /**
+     *  Removes a <code>Task</code> with a given task ID from this <code>TaskList</code> and returns it.
+     *  @param id <code>int</code> ID of the task.
```

Comment:
It would be helpful to specify whether the `ID` is 0-based or 1-based.

Reply:
Thanks for the suggestion - will do so in the documentation since it's unclear right now.